packages/node_modules/@ciscospark/plugin-phone/test/integration/spec/phone.js

```diff
@@ -29,7 +32,7 @@ browserOnly(describe)('plugin-phone', function () {
   this.timeout(60000);
 
   describe('Phone', () => {
-    let mccoy, spock;
+    let mccoy, spock, quark;
```

Comment:
![img](https://media3.giphy.com/media/jkHVh4QbKi8Ja/giphy.gif?cid=e1bb72ff5b748c6647595466733b7db3)

I had to ask around for a good trek name.